ext/handlers/common_test.go

```diff
@@ -33,11 +33,11 @@ func NewTestBot() *gotgbot.Bot {
 	}
 }
 
-func NewMessage(userId int64, chatId int64, message string) *ext.Context {
+func NewMessage(userId, chatId int64, message string) *ext.Context {
```

Comment:
TL;DR Specifying the type after each parameter is intentional, and I wouldn't want to remove them.

IMO, skipping the types might reduce that character count, but it makes the method harder to read - you have to look further than you variable to check.
It also opens up to new bugs, such as if someone were to add a parameter, and unknowingly change other parameter types.

eg `NewMessage(userId, newfield string, chatId int64, message string)` 

Suddenly, userId is a string instead of an int64. 

This would get caught on internal methods with compile errors, but might not be caught on external methods; causing issues down the line.